api/src/routes/index.js

```diff
@@ -1,9 +1,36 @@
-var express = require("express");
-var router = express.Router();
+const express = require("express");
+const { StatusCodes } = require("http-status-codes");
+
+const { getStudentById } = require("../services/student.js");
+
+const router = express.Router();
```

Comment:
Very nitpicky, but do you think we should reorder these imports, moving router under express and the two others immediately below and without a new line between them?

Reply:
Fixed! In general, I think it's still a good idea to separate third-party requires and internal requires with a blank line, though. So we can differentiate them at a glance. I pushed a new commit as an example.

Reply:
```
const express = require("express");
const router = express.Router();
const statusCodes = require("http-status-codes");

const { getStudentById } = require("../services/student.js");

```

Reply:
Yep we should do this instead, thanks for pointing that out :)